skypy/halos/redshift.py

```diff
@@ -0,0 +1,19 @@
+"""Halo redshifts.
+
+This module implements functions to sample from redshift distributions of dark
+matter halos.
+
+Models
+======
+.. autosummary::
+   :nosignatures:
+   :toctree: ../api/
+
+   colossus_mf_redshift
+"""
+
+from ._colossus import colossus_mf_redshift
+
+__all__ = [
+    'colossus_mf_redshift',
+]
```

Comment:
I don't understand the purpose of this file.

Reply:
This mirrors the module structure used for the galaxy luminosity and stellar mass functions